css/properties/cursor.json

```diff
@@ -71,7 +74,8 @@
                 "version_added": "3"
               },
               "safari_ios": {
-                "version_added": "1"
+                "version_added": false,
+                "notes": "If this value is used, the iPad will display the <code>default</code> pointer instead."
```

Comment:
A question for this and the other entries with those note:

You test case shows only `<p>` elements. With just the UA styles, does the cursor change for buttons, inputs, or contenteditable elements? I'm wondering if it's (correctly) falling back to `auto` and not `default`.

If it's the former and not the latter, then these notes aren't necessary.

(Or is this something even weirder, where `CSS.supports("cursor: alias")` returns true but it treats it as `auto` or `default`?)

Reply:
I've tested on other element types as well, and confirmed that the behavior is identical for buttons, inputs, etc. too.

Also, unfortunately it is not defaulting to `auto` and it is specifically falling back to `default`.  The `default` value keeps the cursor as a dot regardless of what it is hovering over, but `auto` will display a text selection cursor (a line) if hovering over any text or text input.  Using a value like `pointer` keeps the cursor as a dot; therefore its behavior is identical to `default`.

Reply:
Ah that's a bummer. OK.